### PR TITLE
Add AnimeOnsen presence

### PR DESCRIPTION
**Pull request by mist8kengas**

### CODE REVIEW

**Review by EncryptedDev — COMMENTED** [review]
Resolve ESLint issues (`yarn lint`)

**Comment by mist8kengas** [comment]
image
Parsing error on two different presences that I do not own, should I ignore?

File: websites/A/AnimeOnsen/presence.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    title: string,
+    episode: number,
+    episodeName: string,
+    playbackState: string
+}
+
+// create new presence
+const presence = new Presence({
+    clientId: "826806766033174568",
+    appMode: true
```
Comment (Slowlife01) [inline websites/A/AnimeOnsen/presence.ts:17]: Deprecated.
```suggestion
```

File: websites/A/AnimeOnsen/presence.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            presenceData.smallImageKey = rpaImage.general.read;
+            presenceData.smallImageText = "Cookies Page";
+            break;
+        }
+
+        default: break;
+    }
+
+    // update presence activity
+    presence.setActivity(presenceData, true);
```
Comment (Slowlife01) [inline websites/A/AnimeOnsen/presence.ts:153]: Not needed
```suggestion
    presence.setActivity(presenceData);
```

File: websites/A/AnimeOnsen/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "color": "#1a1a1a",
+    "tags": [
+        "anime",
+        "video",
+        "streaming"
+    ],
+    "category": "anime",
+    "regExp": "(www.)?(animeonsen.xyz)",
+    "iframe": false,
+    "readLogs": false
```
Comment (Slowlife01) [inline websites/A/AnimeOnsen/dist/metadata.json:24]:
```suggestion
```

**Comment by Slowlife01** [comment]
Show proof of the Presence working afterwards

**Comment by mist8kengas** [comment]
> Show proof of the Presence working afterwards
account
browsing
page-static
page-static-small-hover
searching
watching-paused
watching-playing

**Review by EncryptedDev — DISMISSED** [review]

**Comment by EncryptedDev** [comment]
Please remove those comments; none of them have any validity

**Review by Bas950 — APPROVED** [review]
lgtm

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**